gazebo/common/CommonIface.cc

```diff
@@ -159,7 +162,12 @@ void common::add_search_path_suffix(const std::string &_suffix)
 /////////////////////////////////////////////////
 std::string common::find_file(const std::string &_file)
 {
-  return common::SystemPaths::Instance()->FindFile(_file, true);
+  std::string path = common::FuelModelDatabase::Instance()->ModelPath(_file);
+  if (path.empty())
+  {
+    path = common::SystemPaths::Instance()->FindFile(_file, true);
+  }
+  return path;
 }
 
 /////////////////////////////////////////////////
```

Comment:
All overloads of the `find_file` function should behave the same way, so I think the functions on line 174 and 180 should also be modified to support Fuel.

Reply:
Added some logic https://github.com/osrf/gazebo/pull/2822/commits/176b2df56aaff948475f5d66091e79ac6030e92d